packages/picasso/src/QuillEditor/hooks/useQuillInstance/useQuillInstance.tsx

```diff
@@ -66,22 +92,32 @@ const formats: QuillOptionsStatic['formats'] = [
 const useQuillInstance = ({
   id,
   placeholder,
+  plugins,
 }: EditorOptionsType): Quill | undefined => {
   const [quill, setQuill] = useState<Quill>()
   const typographyClasses = useTypographyClasses()
 
   useEffect(() => {
+    let extendedFormats: QuillOptionsStatic['formats'] = [...formats]
+
     Quill.register(makeHeaderFormat(typographyClasses), true)
     Quill.register(makeBoldFormat(typographyClasses), true)
 
+    const allowLinks = plugins?.includes('link')
+
+    if (allowLinks) {
+      Quill.register(makeLinkFormat(typographyClasses), true)
+      extendedFormats = extendedFormats.concat('link')
```

Comment:
`.push()` should be simpler here, wdyt?
```suggestion
      extendedFormats.push('link')
```

Reply:
Yes, make sense :)